lib/tapioca.rb

```diff
@@ -21,4 +21,5 @@ class Error < StandardError; end
 require "tapioca/constant_locator"
 require "tapioca/compilers/dsl/base"
 require "tapioca/helpers/active_record_column_type_helper"
+require "tapioca/helpers/dynamic_mixin_compiler"
```

Comment:
I don't think this is a helper anymore, it has actually turned into a compiler itself. So maybe it should live under `tapioca/compilers/` too.